TLM/TLM/Manager/Impl/SpeedLimitManager.cs

```diff
@@ -48,19 +52,62 @@ public class SpeedLimitManager
         public static readonly SpeedLimitManager Instance = new();
 
         /// <summary>For each NetInfo name: custom speed limit.</summary>
-        private readonly Dictionary<string, float> customLaneSpeedLimit_ = new();
+        private readonly Dictionary<NetInfo, float> customNetinfoSpeedLimits_ = new();
```

Comment:
Custom default. Isn't custom netinfo not clear enough?
at least its more clear than custom lane! there is doc above that says what it does.
I will see what I can do. In #1275 i should have addressed these issues too but I forgot.

the names in SpeedLimitManager are very confusing in general. Partly because of inconsistent naming and partly because there are too many methods.

Reply:
I think this should be part of another PR as renaming can easily make things out of hand.

Reply:
@aubergine10 in that PR we should consider the difference between:
- Per NetInfo Default speed limit (if customized, it applies to all vehicle lanes).
- Per NetInfo.Lane Default Speed limit  (if not customized, the vanilla value for each lane could be different).
- Generic Default Speed Limit : a method that checks if the speed limit is customised or not and return a value accordingly (GetDefaultGameSpeedLimit in my code)
- Vanilla Default per NetInfo.Lane Speed Limit : I don't think we need a method for this because its simple laneInfo.m_speedLimit (old code was more complicated).

There is also the issue of Units
- SpeedValue (struct)
- Game (1 => 50KPH) (useful return value in performance critical methods)

for each lane instance we have 
- overridden/set lane speed limit in game units.
- There is also the cached final speed limit in game units for all lanes all things considered (overriden/vanilla-default/customsied-default)